plugins/arElasticSearchPlugin/lib/model/arElasticSearchActorPdo.class.php

```diff
@@ -322,4 +374,18 @@ protected function getDirectlyRelatedTerms($typeId)
 
     return self::$statements['relatedTerms']->fetchAll(PDO::FETCH_OBJ);
   }
+
+  protected function getProperty($name)
```

Comment:
We should probably make an issue, as this is shared with arElasticSearchInformationObjectPdo, to refactor this so the code's shared.

Reply:
I agree.  Will see that tickets are created.